src/test/resources/bundles/userCardExamples2/template/usercard_message.handlebars

```diff
@@ -22,6 +22,7 @@
         summary : {key : "message.summary"},
 		title : {key : "message.title"},
         data : {message: message},
+        entitiesAllowedToEdit: ['ALLCONTROLROOMS'],
```

Comment:
i would only touch the defaultProcessV1 process so remove this line and put ALLCONTROLROOMS in src/test/resources/bundles/defaultProcess_V1/template/usercard_message.handlebars